build/debian/postinst

```diff
@@ -7,13 +7,10 @@ if [ -f /usr/share/debconf/confmodule ]; then
     . /usr/share/debconf/confmodule
 fi
 
-for file in node_exporter mysqld_exporter postgres_exporter mongodb_exporter proxysql_exporter
-do
-    ln -s -f /usr/local/percona/$file /usr/bin/$file
-done
 for file in pmm-admin pmm-agent
 do
-    ln -s -f /usr/sbin/$file /usr/bin/$file
+    ln -s -f /usr/local/percona/pmm2/bin/$file /usr/bin/$file
+    ln -s -f /usr/local/percona/pmm2/bin/$file /usr/sbin/$file
```

Comment:
Do we really need both symlinks? Below we use only `sbin`.

Reply:
yes because previously we installed file in /usr/bin/ and now it will be in /usr/local/percona/pmm2/bin